internal/storage/caching/fast_stat_bucket_test.go

```diff
@@ -768,3 +768,18 @@ func (t *StatObjectTest) TestShouldCallGetFolderWhenEntryIsNotPresent() {
 	AssertEq(nil, err)
 	ExpectThat(result, Pointee(DeepEquals(*folder)))
 }
+
+func (t *StatObjectTest) TestShouldRenameFolder() {
```

Comment:
fix the test name. 
why there is should in test when you are not specifying any condition

Reply:
Done